framework/include/utils/MathFVUtils.h

```diff
@@ -90,34 +90,47 @@ faceArgSubdomains(const SubdomainRestrictable & obj, const FaceInfo & fi)
                         makeSidedFace(obj, fi.neighborPtr(), fi).sub_id);
 }
```

Comment:
is this file still MathFVUtils ? Seems more like FaceSomethingFVUtils ?

Reply:
These C++ functions help create arguments for evaluating functions. I consider that mathy